src/platform/forms-system/src/js/helpers.js

```diff
@@ -642,3 +642,28 @@ export function transformForSubmit(
 
   return JSON.stringify(withoutViewFields, replacer) || '{}';
 }
+
+export function showReviewField(
```

Comment:
Please add a JSDoc-style comment to this helper to explain what it does and why. I know that you're just moving this from another location, but the context will be nice to have in the future.

Reply:
Can do!  I'll also be adding some tests as well.